docker/standalone-ha.xml

```diff
@@ -128,8 +132,16 @@
             </formatter>
         </subsystem>
         <subsystem xmlns="urn:jboss:domain:bean-validation:1.0"/>
-        <subsystem xmlns="urn:jboss:domain:datasources:5.0">
+        <subsystem xmlns="urn:jboss:domain:datasources:4.0">
             <datasources>
+                <datasource jndi-name="java:jboss/datasources/ExampleDS" pool-name="ExampleDS" enabled="true" use-java-context="true">
```

Comment:
Thanks for the review, will remove. This came in as part of `distribution/server-dist/target/keycloak-3.2.0.Final/standalone/configuration/standalone-ha.xml`